cli/cmd/plugin/ui/web/tanzu-ui/src/constants/actionTypes.ts

```diff
@@ -0,0 +1,2 @@
+export const TEXT_CHANGE = 'TEXT_CHANGE';
```

Comment:
I'd love to see us start by organizing actions under different headings in this file.

For example:
// Navigation actions
Toggle nav action here

// Form input actions
Text change action here

and we will continue to build on that pattern...eventually separating certain groups of actions out to their own constants files if we find that makes sense